patterns/distributed/roles/apim-control-plane/templates/carbon-home/bin/api-manager.sh.j2

```diff
@@ -0,0 +1,362 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
+#  Copyright 2005-2012 WSO2, LLC. http://www.wso2.org
```

Comment:
Do we need to change this to 2023 WSO2 also? If so there are multiple places with that replacement needed.